<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>690</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statusButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Conexi√≥n USB</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="serialPortLabel">
     <property name="text">
      <string>Puerto Serie:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serialPortComboBox"/>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pingButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
  </widget>
  <widget class="qfi_ADI" name="ElementoRoll">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="qfi_HSI" name="ElementoYaw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>YAW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>419</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ROLL AND PITCH</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>420</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PITCH</string>
   </property>
  </widget>
  <widget class="QwtCompass" name="PitchCompass">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>450</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="drone">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/PITCH.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QwtSlider" name="ControlVelocidad">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>60</width>
     <height>250</height>
    </rect>
   </property>
   <property name="upperBound">
    <double>200.000000000000000</double>
   </property>
   <property name="totalSteps">
    <UInt>200</UInt>
   </property>
   <property name="pageSteps">
    <UInt>1</UInt>
   </property>
  </widget>
  <widget class="QwtDial" name="RuedaVelocidad">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QwtAnalogClock" name="Reloj">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QwtThermo" name="Deposito">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>440</y>
     <width>91</width>
     <height>251</height>
    </rect>
   </property>
   <property name="pipeWidth">
    <number>50</number>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="Counter" name="PanelAltitud">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="digits">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="CristalRoto">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>711</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/Broken-Glass-PNG-Image-Transparent.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VELOCIDAD</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CONTROL DE VELOCIDAD </string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>410</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COMBUSTIBLE (L)</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ALTITUD</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RELOJ DE A BORDO</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Km/h</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Counter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/counter.h</header>
  </customwidget>
  <customwidget>
   <class>qfi_ADI</class>
   <extends>QGraphicsView</extends>
   <header>qfi_ADI.h</header>
  </customwidget>
  <customwidget>
   <class>qfi_HSI</class>
   <extends>QGraphicsView</extends>
   <header>qfi_HSI.h</header>
  </customwidget>
  <customwidget>
   <class>QwtAnalogClock</class>
   <extends>QwtDial</extends>
   <header>qwt_analog_clock.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
